src/store/index.js

```diff
@@ -12,5 +12,31 @@ export default new Vuex.Store({
   // tree: https://vuex.vuejs.org/guide/modules.html
   modules: {
     people
+  },
+  getters: {
+    // vuex.esm.js:95 Uncaught Error: [vuex] getters should be function but "getters.people" is {}.
+    // Getting the error above, so I understand that this isn't proper Vuex syntax.
+    // If I was better with Vue/Vuex syntax I would do something like this.
+    people: {
+      async get (state, id) {
```

Comment:
Getters are really just meant to get and transform state data (they can't be async), what you would want to do is move this fetching logic to an Action which can be dispatched and be async.